### PR TITLE
Fixed typo in time format hh:mm:ss > HH:mm:ss

### PR DESCRIPTION
**Pull request by afanasy**

### CODE REVIEW

File: build/server/konnectors/isen.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ ical = require('cozy-ical');
 
 vcal = require('cozy-ical').VCalendar;
 
-NotificationHelper = require('cozy-notifications-helper');
+NotificationHelper = require('cozy-notifhications-helper');
```
Comment (ZeHiro) [inline build/server/konnectors/isen.js]: nit : notifhication -> notifications

**Comment by ZeHiro** [comment]
Thanks @afanasy Can you also make the modification in the source file : 
https://github.com/cozy-labs/konnectors/blob/master/server/konnectors/isen.coffee#L481

**Comment by nicofrand** [comment]
There is a typo and unfortunately the fix must be done on the coffee file, not the one generated in the build directory. But thanks a lot @afanasy, I'll fix it when https://github.com/cozy-labs/konnectors/pull/462 is merged !

**Comment by afanasy** [comment]
Created another PR

**Pull request closed without merging**